packages/dropdown/src/types.ts

```diff
@@ -30,19 +32,34 @@ type FzDropdownProps = {
    * Class binded to opener
    */
   openerClass?: string
+  /**
+   * teleport floating to body
+   */
+  teleport?: boolean
 }
 
 type FzDropdownSlots = {
   /**
    *
    * Label of the standard button opener
    */
-  default(props: { isOpen: boolean }): any
+  default(props: { isOpen: boolean }): VNode | VNode[]
   /**
    *
    * Use this to replace the button opener entirely
    */
-  opener(props: { isOpen: boolean }): any
+  opener(props: { isOpen: boolean }): VNode | VNode[]
+}
+
+interface FzIconDropdownProps extends FzDropdownProps {
```

Comment:
I'm pretty sure that storybook isn't able to document props automatically this way

Reply:
I tried to make it a `type`, but it just refuses to compile in storybook. I tried in TS playground and works just fine, but here it breaks the compilation, no idea why